### DISCUSSION TITLE
My drone's M3 port for the motor is not working

### DISCUSSION THREAD

**Discussion by akshaysunkara** [bitcraze/discussions#1432] [Q&A]
Hey guys, I'm having some issues with the m3 port of my crazyflie 2.1 drone:

Basically, whenever I attach any motor with a propellor on it, it automatically starts moving slowly, not rapidly.

Whenever I run a propellor test on it, it gives me a variance of 0 for that specific motor. 

I've tested it with many different motors but its the same thing again and again.

Also, the port seems to be semi-detached like in this picture but can be put back to the board by just pressing it below.

![Untitled](https://github.com/bitcraze/discussions/assets/171630750/7b106b28-eea0-499f-8f14-8c19eee6f460)

Any fixes? I really don't wanna purchase a whole other drone.

**Comment by tobbeanton** [comment]
Have you inspected the connector? With a soldering iron you (or someone that can help you) will most likely be able to fix it if it is a bad/broken connection.

**Reply by akshaysunkara** [reply]
What do I have to inspect?

Right now the port is just hanging a little off the board, the end is connected though.

How can I solder it to connect again?

**Reply by tobbeanton** [reply]
When you say it is hanging, is it bent or broken? Perhaps you could take a close-up picture of the connector from the back?

> How can I solder it to connect again?

If you don't have any solder experience perhaps you know someone that has?

**Reply by akshaysunkara** [reply]
When I say that this is hanging, I mean to say that the port where you connect the motor's wires to isn't attached to the board anymore.

![Untitled](https://github.com/bitcraze/discussions/assets/171630750/4f49096d-bf9e-492f-9b0a-9bfd12a2aa1b)

In the picture the port is hanging from the back, still connected, just very loosely. 

Here are more pictures of the port:

![Untitled](https://github.com/bitcraze/discussions/assets/171630750/da89efc7-00f1-4659-a756-ee2c352365ce)
![Untitled](https://github.com/bitcraze/discussions/assets/171630750/de70dca5-977c-4b0e-a940-44cf3a7b6e2a)
![Untitled](https://github.com/bitcraze/discussions/assets/171630750/bf963307-bc67-4033-9f67-b70a4b315fe4) 

This might not be the reason for the issue I described in the first post but it's the only thing I can think of.

I can find someone who has experience soldering, I just need to tell them exactly what to solve or solder.

Thanks a lot again.

**Comment by tobbeanton** [comment]
It looks a bit weird, like the solder joints has come loose. Try re-melting the solder on the connector solder side and push in the connector. Thus one would have to melt booth solder contacts at the same time.
